website/docs/r/compute_ssl_certificate.html.markdown

```diff
@@ -22,6 +22,29 @@ resource "google_compute_ssl_certificate" "default" {
   description = "a description"
   private_key = "${file("path/to/private.key")}"
   certificate = "${file("path/to/certificate.crt")}"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+# You may also want to control name generation explicitly:
+
+resource "random_id" "certificate" {
+  byte_length = 4
+  prefix      = "my-certificate-"
```

Comment:
This would actually not work as intended, I believe--I think we need to use [keepers](https://www.terraform.io/docs/providers/random/index.html#resource-quot-keepers-quot-) for this to generate a new random ID.

Reply:
@paddycarver thx for catching it, fixed!  (or at least I hope i did :) )